src/web/src/components/SearchBar.tsx

```diff
@@ -20,14 +20,24 @@ const useStyles = makeStyles((theme: Theme) =>
     root: {
       overflow: 'visible',
       maxWidth: '785px',
-      padding: 0,
       marginLeft: 'auto',
       marginRight: 'auto',
+      padding: theme.spacing(2, 0, 0, 0),
       marginBottom: theme.spacing(6),
     },
     card: {
-      padding: theme.spacing(2, 4, 2, 4),
-      backgroundColor: theme.palette.background.default,
+      padding: theme.spacing(0, 0, 0, 2),
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      borderRadius: '50px',
+      background: '#d1d1d1',
+      transition: 'background-color .5s',
+      '&:hover': {
+        backgroundColor: '#ffffff',
+        border: 'solid 1px',
```

Comment:
Can we add a `border: solid 1px transparent` when not hovered so it doesn't jump?
![outline-bug](https://user-images.githubusercontent.com/427398/109391216-f481f900-78e3-11eb-8a75-7fcebbc8d9b2.gif)

Reply:
Done. I will tweak this for mobile responsiveness